hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/operations/ConfigMismatchOp.java

```diff
@@ -57,6 +57,13 @@ protected void readInternal(ObjectDataInput in) throws IOException {
     public void run() {
         NodeEngineImpl nodeEngine = (NodeEngineImpl) getNodeEngine();
         Node node = nodeEngine.getNode();
+
+        // Ignore shutdown requests if we are already in a cluster, as being
+        // part of a cluster means our config was already verified & accepted
+        if (node.getClusterService().isJoined()) {
+            return;
```

Comment:
Can you add logging here also?

Reply:
Done, let me know if you were looking for something different 👍